changelog/fragments/1668591286-fix-beats-logs.yaml

```
@@ -0,0 +1,36 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: bug-fix
+
+# Change summary; a 80ish characters long description of the change.
+summary: "monitoring: fix JSON logs parsing for Beats"
+
+# Long description; in case the summary is not enough to describe the change
+# this field accommodate a description without length limits.
+description: >-
+  This commit fixes the JSON parsing from Beats logs. The configuration sent to
```

[Suggestion]
I believe "this commit" isn't the best way to describe a change. I'd suggest "This change" or "this fix".
```suggestion
  Fixes the JSON parsing from Beats logs. The configuration sent to
```